client/components/tinymce/plugins/insert-menu/menu-items.jsx

```diff
@@ -11,38 +11,38 @@ import i18n from 'i18n-calypso';
 import config from 'config';
 
 /* eslint-disable wpcalypso/jsx-classname-namespace */
-const GridiconButton = ( { icon, label } ) => (
+const GridiconButton = ( { icon, label, insert } ) => (
```

Comment:
hmm I suppose we could leave it if we think we might reuse it for something else later